Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -320,6 +348,34 @@ void webkit_user_content_manager_unregister_script_message_handler(WebKitUserCon
     manager->priv->userContentController->removeUserMessageHandlerForName(String::fromUTF8(name), API::ContentWorld::pageContentWorld());
 }
 
+/**
+ * webkit_user_content_manager_register_script_message_handler_with_reply:
+ * @manager: A #WebKitUserContentManager
+ * @name: Name of the script message channel
+ * @world_name: the name of a #WebKitScriptWorld
+ *
+ * Registers a new user script message handler in script world.
+ *
+ * Registers a new user script message handler in script world with name @world_name.
+ * If NULL is passed as the world_name, the default world will be used.
+ * See webkit_user_content_manager_register_script_message_handler() for full description.
+ *
+ * Registering a script message handler will fail if the requested
+ * name has been already registered before.
+ *
```

Comment:
Maybe I would also add a note here telling that handlers which were configured with this function can be unregistered with the existing `webkit_user_content_manager_unregister_script_message_handler()` and `webkit_user_content_manager_unregister_script_message_handler_in_world()` functions&mdash;maybe users will expect an `_unregister_script_message_handler_with_reply()` which does not exist, and mentioning the existing functions we can nudge them in the right direction.